### PR TITLE
Some improvements 👀

### PR DESCRIPTION
**Pull request by ForceUpdate1**
fix particle remaps 1.12 -> 1.13
fix use bed 1.13 -> 1.14
fix npe in piston handler

### CODE REVIEW

File: core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_13_2to1_14/packets/EntityPackets1_14.java [hunk 1]
```diff
@@ -463,11 +464,31 @@ protected void registerRewrites() {
...
         });
 
-        // Remove bed location - todo send sleep packet
+        // Remove bed location
         registerMetaHandler().filter(EntityType1_14.EntityType.LIVINGENTITY, true).handle(e -> {
             Metadata meta = e.getData();
             int index = e.getIndex();
             if (index == 12) {
+                Position position = (Position) meta.getValue();
+                if(position != null){
```
Comment (MylesIsCool) [inline core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_13_2to1_14/packets/EntityPackets1_14.java]: Needs auto format to have same spacing as other if's

File: core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_13_2to1_14/packets/EntityPackets1_14.java [hunk 2]
```diff
@@ -463,11 +464,31 @@ protected void registerRewrites() {
...
         });
 
-        // Remove bed location - todo send sleep packet
+        // Remove bed location
         registerMetaHandler().filter(EntityType1_14.EntityType.LIVINGENTITY, true).handle(e -> {
             Metadata meta = e.getData();
             int index = e.getIndex();
             if (index == 12) {
+                Position position = (Position) meta.getValue();
+                if (position != null) {
```
Comment (MylesIsCool) [inline core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_13_2to1_14/packets/EntityPackets1_14.java]: Would be nice to have a comment stating what these packets are

**Comment by KennyTV** [comment]
Rebase > Merge 🐱‍🐉

**Comment by MylesIsCool** [comment]
Ty

**Pull request merged**